lib/glimesh/accounts.ex

```diff
@@ -315,6 +315,57 @@ defmodule Glimesh.Accounts do
     |> Repo.update()
   end
 
+  def change_tfa(user, attrs \\ %{}) do
+    User.tfa_changeset(user, attrs)
+  end
+
+  def update_tfa_temp(user, attrs) do
+    user
+    |> User.tfa_changeset(attrs)
+    |> Repo.update()
+  end
+
+  def update_tfa(user, pin, attrs) do
+    secret = user.tfa_token
+    IO.puts("attrs #{attrs.tfa_token}")
+    IO.puts("user #{user.tfa_token}")
```

Comment:
Remove for prod merge

Reply:
whops seems I forgot some debugging indeed XD